### PR TITLE
better mini walking policy

### PR DESCRIPTION
**Pull request by wuallenwu**

### CODE REVIEW

File: sim/humanoid_gym/play_h5.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
+import argparse
```
Comment (budzianowski) [inline sim/humanoid_gym/play_h5.py:1]: remove this

Comment (wuallenwu) [reply sim/humanoid_gym/play_h5.py]: addressed

**Comment by budzianowski** [comment]
Can we test:
`        joint_diff = self.dof_pos - self.default_joint_pd_target
        left_yaw_roll = joint_diff[:, [self.legs_joints["left_hip_roll"], self.legs_joints["left_hip_yaw"]]]
        right_yaw_roll = joint_diff[:, [self.legs_joints["right_hip_roll"], self.legs_joints["right_hip_yaw"]]]
        yaw_roll = torch.norm(left_yaw_roll, dim=1) + torch.norm(right_yaw_roll, dim=1)
        yaw_roll = torch.clamp(yaw_roll - 0.1, 0, 50)
        return torch.exp(-yaw_roll * 100) - 0.01 * torch.norm(joint_diff, dim=1)
`
